concepticondata/concepticon.tsv

```diff
@@ -3174,3 +3174,55 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3179	CORAL ROCK	The physical world	Limestone created from the skeleton of reef-forming stony corals (Scleractinia), displaying their growth patterns.	Person/Thing	
 3180	SHOOT (WITH SLINGSHOT)	Warfare and hunting	Use a rubber-based projectile weapon to propel projectiles against a target.	Action/Process	
 3181	MUSTARD	Food and drink	It can be a yellow sauce with a strong taste or a plant which is used to make mustard sauce.	Person/Thing	
+3182	MARRIAGE OR WEDDING	Kinship	The ceremony of marrying someone or the state of being married.	Person/Thing	
+3183	BREATH OR BREATHE	The body	The air that goes into one's lungs or the act of breathing	Other	
+3184	BE ALIVE OR LIFE	The body	The act of being alive or the life itself.	Other	
+3185	BE DEAD OR DIE OR KILL	The body	To be not alive or to cease to live or to cause the end of a life.	Other	
+3186	BE DEAD OR DEATH OR DIE	The body	To be not alive or the end of a life or to cease to live.	Other	
+3187	DROWNED	The body	To have perished by suffocation in water.	Property	
+3188	MIGHTY OR POWERFUL OR STRONG	Emotions and values	Holding much (political, social, ...) power or being done using a huge amount of energy or having great physical strength.	Property	
+3189	STAB TO DEATH	The body	To kill by stabbing with a knife.	Action/Process	
+3190	BROKEN (NOT WORKING)	Basic actions and technology	Not capable of working.	Property	
+3191	BROKEN (SHATTERED)	Basic actions and technology	Broken into pieces.	Property	
+3192	BRONZE OR COPPER	The physical world	Chemical element with symbol Cu and atomic number 29 or an alloy made from it.	Person/Thing	
+3193	PAINT (WALL)	Basic actions and technology	To apply color to a wall.	Action/Process	
+3194	PAINT (WALL) OR PAINTING	Basic actions and technology	To apply color to a wall or to make a painting or drawing.	Action/Process	
```

Comment:
Do we have wordlists which actually make this distinction, between 3193 and 3194?

Reply:
also, be careful with the gloss: paint a wall is usually not meant as "to paint a real wall", but to distinguish "paint (something like a picture)" from "paint (=color something)". Taking elicitation glosses to literal is tricky: we need to understand why people elicit them. And here, I don't see that a concept is THAT specific.

Reply:
> Do we have wordlists which actually make this distinction, between 3193 and 3194?

Buck makes the distinction between PAINT (WALL) ("paint a house") and PAINTING ("paint as artist").
However, I now noticed we already have PAINT (SOMETHING) which is what I meant by PAINT (WALL). I'll remove that new concept and relink all I linked to it to PAINT (SOMETHING) and also adjust the hyperconcept.